jpa/subsystem/src/main/java/org/jboss/as/jpa/processor/secondlevelcache/InfinispanCacheDeploymentListener.java

```diff
@@ -92,25 +94,19 @@ public Wrapper startCache(Classification classification, Properties properties)
                 builder.requires(dependencyName);
             }
         }
-
+        final UninterruptibleCountDownLatch latch = new UninterruptibleCountDownLatch(1);
```

Comment:
Is there a particular reason for ignoring interrupts here?  Wouldn't it be more appropriate to propagate the InterruptedException to the caller?

Reply:
No particular reason - fixed.